config/site/navigation.config.js

```diff
@@ -17,11 +18,16 @@ const navConfig = {
       path: '/components',
       text: 'Components',
       pageTypes: ['doc'],
+      capabilities: {
+        devToolbar: true,
+      },
     }, {
       path: '/tests',
       text: 'Tests',
       pageTypes: ['test'],
-      isHidden: true,
+      capabilities: {
+        devToolbar: true,
+      },
     }],
```

Comment:
Could we add the test evidence link to the nav config instead of automatically injected in later on? We could still use the `includeTestEvidence` to toggle adding the link or not

Reply:
I'd like to leave evidence as is. I regret making the change to add evidence to dev site and i don't think it belongs here, so i don't want to touch it.

Reply:
Sure thing. I agree the evidence just bloats the site assets and slows down compile so it's not my favorite feature of this. Do we have ideas for removing/replacing them?